test/run

```diff
@@ -249,6 +261,20 @@ test_from_dockerfile(){
   check_result $?
 }
 
+test_from_dockerfile_minimal(){
+  info "Test from Dockerfile"
+
+  # uwsgi in uwsgi-test-app
+  ct_test_app_dockerfile $test_dir/from-dockerfile/uwsgi.Dockerfile.tpl $test_dir/uwsgi-test-app 'Hello World from uWSGI hosted WSGI application!' app-src
+  check_result $?
+
+  # mod_wsgi in micropipenv-requirements-test-app
+  # TODO: Find a better way how to specify full/minimal images that should work together
+  sed "s@#IMAGE_NAME#@${IMAGE_NAME}@;s@#FULL_IMAGE_NAME#@ubi{{ spec.el_version }}/python-{{ spec.short_ver }}:1@" $test_dir/from-dockerfile/mod_wsgi.Dockerfile.tpl > $test_dir/from-dockerfile/Dockerfile
+  ct_test_app_dockerfile $test_dir/from-dockerfile/Dockerfile $test_dir/micropipenv-requirements-test-app 'Hello World from mod_wsgi hosted WSGI application!' app-src
+  check_result $?
+}
+
```

Comment:
`test_from_dockerfile` and `test_from_dockerfile_minimal` look very different, can you explain why?

Reply:
Both tests call ct_test_app_dockerfile twice for two applications - where do you see a big difference?

Reply:
Added a comment suggestion instead.